shaman/controllers/repos/projects.py

```diff
@@ -55,12 +55,17 @@ def index_post(self):
             url=request.json.get("url", ""),
         )
         repo.update_from_json(update_data)
+        archs = request.json.get("archs", [])
+        for arch in archs:
+            data = dict(name=arch, repo=repo)
+            models.get_or_create(Arch, **data)
```

Comment:
it doesn't look like `data` is used anywhere else, this can consume `name` and `repo` directly